src/utils/parse_utils.ts

```diff
@@ -24,9 +24,12 @@ export const parseUtils = {
         request: ParseRequestForExcludedSourcesParams,
         validApiKeys: string[],
         endpoint: 'price' | 'quote',
-    ): { excludedSources: ERC20BridgeSource[]; nativeExclusivelyRFQT: boolean } {
+    ): { excludedSources: ERC20BridgeSource[]; includedSources: ERC20BridgeSource[]; nativeExclusivelyRFQT: boolean } {
```

Comment:
This return type has grow a little, have you considered making it it's own type?